.github/workflows/release-compose.yml

```
@@ -0,0 +1,80 @@
+      - name: Get Previous Tag
+        id: get_previous_tag
+        run: |
+          ALL_TAGS=$(git tag --sort=-v:refname)
+          CURRENT_TAG=${GITHUB_REF#refs/tags/}
+          PREVIOUS_TAG=$(echo "$ALL_TAGS" | grep -F -x -A 1 "$CURRENT_TAG" | tail -n 1)
+          echo "previous_tag=$PREVIOUS_TAG" >> $GITHUB_ENV
+
+      - name: Generate Release Notes
+        id: generate_release_note
+        if: env.release_exists == 'false'
+        run: |
```

actionlint (1.7.4)

shellcheck reported issue in this script: SC2181:style:3:7: Check exit code directly with e.g. 'if mycmd;', not indirectly with $?

(shellcheck)